src/Microsoft.ML.Transforms/Text/WordTokenizeTransform.cs

```diff
@@ -35,11 +35,10 @@ public interface ITokenizeTransform : IDataTransform
     {
     }
 
-    /// <summary>
-    /// The input for this transform is a DvText or a vector of DvTexts, and its output is a vector of DvTexts,
-    /// corresponding to the tokens in the input text, split using a set of user specified separator characters.
-    /// Empty strings and strings containing only spaces are dropped.
-    /// </summary>
+    // The input for this transform is a DvText or a vector of DvTexts, and its output is a vector of DvTexts,
```

Comment:
:bulb: Avoid copy-paste documentation text. Something will get out-of-date otherwise and it's almost impossible to figure out what.